book/002-variables.md

```diff
@@ -65,19 +65,19 @@ Before ES2015, aka ES6, there was ES5, the older version of Javascript.
 
 The only way to declare a variable in this version was with using `var`.
 
-`var` is a little quirky, because it's scope lives at the `function` level. It can also be redeclared.
+`var` is a little quirky, because its scope lives at the `function` level. It can also be redeclared.
 
 ```js
-var i = 'oh dang';
+var x = 'oh dang';
 
 function MyFunction() {
-  var i = 10;
-  var i = 1;
-  var i = false;
+  var x = 10;
+  var x = 1;
+  var x = false;
 }
 ```
 
-What's the value of `i`? I don't really know this example... What do?
+What's the value of `x`? Well, at the end of the function `MyFunction()`, `x` is `false`. But after the function block ends, `x` is `'oh dang'`. This is ambiguous behavior and should be avoided when possible.
```

Comment:
Your contribution is valuable here. Thank you!

However, I feel the language here is a little shaky. I try to stay away from starting sentences with "Well," or "so" because it makes the declaration/argument much more definitive.  Let's rephrase this sentence:

```
What's the value of `x`? At the end of `MyFunction()`, `x` is `false`, but outside of `MyFunction()`, `x` is `'oh dang'`. This ambiguous behavior should be avoided as much as possible.